src/libs/runtime-utils.js

```diff
@@ -261,11 +261,20 @@ export const getCurrentAttachedDataDisk = (app, appDataDisks) => {
 }
 
 // If the disk was attached to an app, return the appType. Otherwise return undefined.
-export const getDiskAppType = disk => {
-  const saturnApp = disk.labels.saturnApplication
+export const getDiskAppType = ({ labels: { saturnApplication } }) => {
```

Comment:
I went down this route of unpacking in my previous PRs and found myself not liking it. I think the API is much clearer when it takes a disk.

(I also thought the unpacking failed when saturnApplication is undefined, but I have a unit test for that case, so presumably the method must be working.)